crypto/fipsmodule/CMakeLists.txt

```diff
@@ -354,6 +354,33 @@ if((ARCH STREQUAL "aarch64") AND UNIX)
 
 endif()
 
+if((ARCH STREQUAL "x86_64") AND UNIX AND NOT MY_ASSEMBLER_IS_TOO_OLD_FOR_AVX)
+
+  # Set the source directory for s2n-bignum assembly files
+  set(MLKEM_NATIVE_DIR "${AWSLC_SOURCE_DIR}/crypto/fipsmodule/ml_kem")
+
+  set(MLKEM_NATIVE_X86_64_ASM_SOURCES
+
+    ${MLKEM_NATIVE_DIR}/mlkem/native/x86_64/src/intt.S
+    ${MLKEM_NATIVE_DIR}/mlkem/native/x86_64/src/ntt.S
+    ${MLKEM_NATIVE_DIR}/mlkem/native/x86_64/src/mulcache_compute.S
+    ${MLKEM_NATIVE_DIR}/mlkem/native/x86_64/src/nttfrombytes.S
+    ${MLKEM_NATIVE_DIR}/mlkem/native/x86_64/src/ntttobytes.S
+    ${MLKEM_NATIVE_DIR}/mlkem/native/x86_64/src/nttunpack.S
+    ${MLKEM_NATIVE_DIR}/mlkem/native/x86_64/src/reduce.S
+    ${MLKEM_NATIVE_DIR}/mlkem/native/x86_64/src/tomont.S
+    ${MLKEM_NATIVE_DIR}/mlkem/native/x86_64/src/polyvec_basemul_acc_montgomery_cached_asm_k2.S
+    ${MLKEM_NATIVE_DIR}/mlkem/native/x86_64/src/polyvec_basemul_acc_montgomery_cached_asm_k3.S
+    ${MLKEM_NATIVE_DIR}/mlkem/native/x86_64/src/polyvec_basemul_acc_montgomery_cached_asm_k4.S
+    ${MLKEM_NATIVE_DIR}/mlkem/native/x86_64/src/rej_uniform_asm.S
+  )
+
+  list(APPEND BCM_ASM_SOURCES ${MLKEM_NATIVE_X86_64_ASM_SOURCES})
+
+  set(S2N_BIGNUM_INCLUDE_DIR "${AWSLC_SOURCE_DIR}/third_party/s2n-bignum/s2n-bignum-imported/include")
```

Comment:
This is already defined up on line 200 https://github.com/aws/aws-lc/blob/df96818a4ca4eb3cb28da3d1d7f573f392c7b03d/crypto/fipsmodule/CMakeLists.txt#L200